frontend-react/src/components/FileHandlers/FileHandler.tsx

```diff
@@ -8,14 +8,41 @@ import { WatersPost } from "../../network/api/WatersApiFunctions";
 import { Destination } from "../../resources/ActionDetailsResource";
 import Spinner from "../Spinner"; // TODO: refactor to use suspense
 
-import { FileErrorDisplay, FileSuccessDisplay } from "./FileHandlerMessaging";
+import {
+    FileErrorDisplay,
+    FileSuccessDisplay,
+    FileWarningDisplay,
+} from "./FileHandlerMessaging";
 import { FileHandlerForm } from "./FileHandlerForm";
 
 // values taken from Report.kt
 const PAYLOAD_MAX_BYTES = 50 * 1000 * 1000; // no idea why this isn't in "k" (* 1024).
 const REPORT_MAX_ITEMS = 10000;
 const REPORT_MAX_ITEM_COLUMNS = 2000;
 
+const SERVER_ERROR_MESSAGING = {
+    heading: "Error",
+    message: "There was a server error. Your file has not been accepted.",
+};
+
+const errorMessagingMap = {
```

Comment:
Is this stuff that ends up coming back from the new validator / hl7 uploader endpoints down the line?